Build breakage because of xml-rs change

The build is broken because xml-rs added a new field to its config (in https://github.com/netvl/xml-rs/commit/03c7c3e746cd524f6d50c6007abd13f7989fffa3).

    error[E0063]: missing field `extra_entities` in initializer of `xml_rs::ParserConfig`
      --> src/xml/reader.rs:25:22
       |
    25 |         let config = ParserConfig {
       |                      ^^^^^^^^^^^^ missing `extra_entities`

Arguably, the xml-rs change did not follow semver, they should have bumped minor. In any case, looks like the fix is to add an empty hashmap.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,7 @@ serde_serialization = ["serde"]
 
 [dependencies]
 rustc-serialize = "0.3.19"
-xml-rs = "0.3.4"
+xml-rs = "0.3.8"
 byteorder = "0.5.3"
 chrono = "0.2.22"
 serde = { version = "0.9.5", optional = true }

--- a/src/xml/reader.rs
+++ b/src/xml/reader.rs
@@ -3,6 +3,7 @@ use chrono::format::ParseError as ChronoParseError;
 use rustc_serialize::base64::FromBase64;
 use std::io::Read;
 use std::str::FromStr;
+use std::collections::HashMap;
 use xml_rs::reader::{EventReader as XmlEventReader, ParserConfig, XmlEvent};
 
 use {Error, Result, PlistEvent};
@@ -28,6 +29,7 @@ impl<R: Read> EventReader<R> {
             cdata_to_characters: true,
             ignore_comments: true,
             coalesce_characters: true,
+            extra_entities: HashMap::default(),
         };
 
         EventReader {

